Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -21,7 +21,6 @@ namespace AzToolsFramework::ViewportUi::Internal
 {
     const static int HighlightBorderSize = 5;
     const static char* HighlightBorderColor = "#4A90E2";
```

Comment:
Hmm I don't believe it's globally accessible, this is static in the .cpp so we have internal linkage (I know it can leak out with Unity builds but the chance of misuse is low). I can still add this for sure but not something that's likely to cause issues